src/wasm/jump-table-assembler.cc

```diff
@@ -408,6 +408,46 @@ void JumpTableAssembler::NopBytes(int bytes) {
   }
 }
 
+#elif V8_TARGET_ARCH_RISCV64
```

Comment:
Duplicate with line 371? Or should be RISCV32?

Reply:
done